nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -290,31 +323,38 @@ define([
                 tabs: {
                     configure: {
                         label: 'Configure',
-                        widget: ConfigureWidget
+                        widget: ConfigureWidget,
+                        class: true
```

Comment:
Some of the widgets are defined in classes and come are defined as factory functions. You can't initialize them the same way, so I put a flag on each tab. The class attribute will need to be updated as new widgets are added.

Reply:
The other widgets in the app cell are all defined as factory functions. I have a slight preference to the class style, but not as strong as my preference for consistency.

After thinking about it some more, we should probably just keep widgets as factories. I can convert the configure tab before it gets merged.

Reply:
Ok, everything is set to be a factory pattern now.

Reply:
I just updated everything to work with the factory pattern :)